### ISSUE TITLE
Can you freeze your packages?

### ISSUE THREAD

**Issue by fabiofumarola** [llSourcell/How_to_make_a_text_summarizer#5]
```
pip freeze > requirements.txt
```

it would be useful to provide a `requirements.txt` to restore the used dependencies.

**Comment by LRParser** [comment]
Agreed! You can find what I got to (partially) work here, @fabiofumarola 
https://github.com/LRParser/How_to_make_a_text_summarizer

**Comment by fabiofumarola** [comment]
Hi @LRParser can you do a pull request. I'm sure @llSourcell will be welcome to accept it
